omod/src/main/webapp/resources/htmlforms/KP_PP_Classification_Tool.xml

```diff
@@ -43,11 +43,284 @@
         form .lablex {
         width: 93%;
         }
-        
+
 
     </style>
+    <script type="text/javascript">
+        if(jQuery){
+          jQuery(document).ready(function(){
+                <ifMode mode="ENTER">
```

Comment:
1. Can you also have the final classification be disabled this way on both Enter and Edit mode.

2. When in edit mode, there is need to enable any of the checkboxes which have values selected as they will be loaded with values and already checked